Sources/KituraNet/HTTP/HTTPServer.swift

```diff
@@ -684,3 +702,7 @@ enum KituraWebSocketUpgradeError: Error {
     // Unknown upgrade error
     case unknownUpgradeError
 }
+
+public enum HTTPServerError: Error {
```

Comment:
Please can you make this an open enum (struct with static cases) as it's likely to be extended in the future